base9/Settings.cpp

```diff
@@ -521,11 +518,10 @@ template<typename T> T Settings::getOrRequest (Node & n, string && f)
     {
         string input = "";
         cout << "Please enter your desired setting for field: '" + f + "':\n";
-        getline(cin, input);
-        cout << "Field '" + f + "' set to: " << input << endl;
```

Comment:
Sounds good, thanks.  I'll start looking at / working on logging the YAML/entered variables.  Should the feature be optional?  And/or should we let the user choose where to log to (either appending to the final output file as you recommended, or logging to a file of user's choice)?